app/src/main/java/org/simple/clinic/di/AppModule.kt

```diff
@@ -160,4 +163,9 @@ class AppModule(private val appContext: Application) {
       appContext: Application,
       configReader: ConfigReader
   ): MinimumMemoryChecker = RealMinimumMemoryChecker(appContext)
+
+  @Provides
+  @TypedPreference(DataProtectionConsent)
+  fun hasUserConsentedToDataProtection(rxSharedPreferences: RxSharedPreferences): Preference<Boolean> =
+      rxSharedPreferences.getBoolean("data_protection_consent")
```

Comment:
This needs to be accessed from the home screen as well, for displaying dialog for existing users. That's why I placed it in the app module rather than creating a new module or scoping to a specific feature.